### ISSUE TITLE
Apply Merge View Fix for Best Fit Line to Filtered Merge View

### ISSUE THREAD

**Issue by cbradyatinquire** [MathNetDev/MathNetClient#17]
"As a teacher i want best fit lines to work in the filtered merge view, when lines are selected."

When a construction has a best fit line, the merge view works in the Normal Merge View.  Find out what the strategy is, and apply this to the Filtered Merge View.

NOTE/IDEA:  In the case where points are not selected to be merged, but lines are, it's ok as a first pass for there to be an error.  As a second pass, we'll want to solve that problem, either by (a)sending the points, or (b) by suppressing the best fit line when points aren't sent, or (c) by another strategy you think ok.

**Comment by mtstevens** [comment]
Tested within MathNetTest, 07/20/18 @ 9:53am PDT. Selecting points and lines seems to resolve the error:

As a teacher, I was able to use Filtered Merge View and move points around without errors, both in 'live' and 'not live' modes.

As a student, I was able to move points around and see that change reflected in Filtered Merge View, all without errors.

**Comment by mtstevens** [comment]
Tested within MathNetDev, 0727/18 @ 7:40am PDT. No errors were encountered.

As a teacher, I was able to successfully select either points or lines, or both, within Filtered Merged View without encountering any errors. I was able to move both points and lines around, in both 'live' and 'not live' modes, without encountering any errors.

As a student, I was still able to move points around and see that change reflected in Filtered Merged View without encountering any errors.
